### ISSUE TITLE
Feature request: Get the date a file was first added to version control

### ISSUE THREAD

**Issue by oobleck** [davidquinn/psi-header#57]
Would be useful in situations when creation date date isn't resolvable.

In Git, this command will provide that date. This will span any subsequent removals and re-adds to the repo:
`git log --diff-filter=A --follow --format=%aI -- <fname> | tail -1`

[explanation](https://stackoverflow.com/a/25633731)

**Comment by oobleck** [comment]
I should also say this is a rad extension and thank you for creating it. 😄

**Comment by davidquinn** [comment]
I will take a look if the VSCE API supports getting this, but last time I tried hooking into it, it extended the execution time of the extension enough that VSC disabled the extension. Maybe it has improved.

**Comment by PierBJX** [comment]
Yes, I agree. It would be nice to have `filecreated` based on the first added version control date instead of based on the clone date. If you need support, I can contribute no problem :)
